src/Operators/pointwisestencil.jl

```diff
@@ -422,22 +421,20 @@ function compose_stencils_at_idx(i_vals_tuple, stencil1, stencil2, loc, idx)
     coefs1 = getidx(stencil1, loc, idx)
     lbw1 = bandwidths(eltype(stencil1))[1]
     lbw, ubw = composed_bandwidths(stencil1, stencil2)
-    j_vals = (1:(ubw - lbw + 1)...,)
     i_func_at_j(j) =
         i -> coefs1[i - lbw1 + 1] ⊠ getidx(stencil2, loc, idx + i)[j - i + lbw1]
     function j_func(j)
         i_vals = i_vals_tuple[j]
         return length(i_vals) == 0 ? zero(eltype(eltype(stencil1))) :
-               return ⊞(map(i_func_at_j(j), i_vals)...)
+               return mapreduce(i_func_at_j(j), ⊞, i_vals)
     end
-    return StencilCoefs{lbw, ubw}(map(j_func, j_vals))
+    return StencilCoefs{lbw, ubw}(ntuple(j -> j_func(j), (ubw - lbw + 1)))
```

Comment:
@charleskawczynski shouldn't we use the same form as we do in the previous stencil code?  this is a `tuplemap` like we had previously and the backing `map` was able to be optimized through in 1.7+.  The real gain was getting rid of the splatting like we discussed.

Reply:
Hmm, maybe you're right, I thought I had tried on just the splatting and didn't see much difference, but I don't remember exactly when the difference in allocations came through.

I'd be happy to continue working on this with you if you'd like. It'd be nice if we can eliminate the remaining allocations, for good measure.